src/main/java/sirius/biz/storage/layer3/MutableVirtualFile.java

```diff
@@ -117,6 +117,18 @@ public MutableVirtualFile withDescription(String description) {
         return this;
     }
 
+    /**
+     * Provides an additional description, containing html tags, which can be shown in the web based UI.
+     *
+     * @param htmlDescription the description to show
+     *                    (this will be {@link sirius.kernel.nls.NLS#smartGet(String) smart translated}).
+     * @return the file itself for fluent method calls
```

Comment:
Is this auto-formatted?

Reply:
Just checked. Somehow it formats it like this

Reply:
Hm, it really looks strange. It's worth checking out the IDE setting again (you did when you checked out the oxomi project, perhaps is missing in sirius-biz)

Reply:
Found the issue, should all be fixed now